src/Components/Admin/AdminAuth.js

```diff
@@ -22,12 +22,23 @@ class AdminAuth extends React.Component {
     this.state = {
       username: "", //stores username
       password: "", //stores password
-      loginError: false,
+      loginError: false
     };
 
     this.fillUsername = this.fillUsername.bind(this);
     this.fillPassword = this.fillPassword.bind(this);
     this.authenticate = this.authenticate.bind(this);
+    this.setShow = this.setShow.bind(this);
```

Comment:
We need to start considering changing stuff to arrow functions to avoid binding `this` every single time. Perhaps a future refactor?

Reply:
Change done for setShow throughout the project. Good catch, it was a small function we could just do inline.